src/components/toast/global_toast_list.tsx

```diff
@@ -48,13 +52,25 @@ export interface EuiGlobalToastListProps extends CommonProps {
    * Determines which side of the browser window the toasts should appear
    */
   side?: ToastSide;
+  /**
+   * Displays a "Clear all" button at the bottom of the toast list
+   * that allows users to dismiss all toasts in a single click.
+   */
+  showClearAllButton?: boolean;
```

Comment:
One approach we could take with this prop that would tie in nicely to the above question about a customizable threshold:

```suggestion
  /**
   * At this threshold, a "Clear all" button will display at the bottom of the toast list
   * that allows users to dismiss all toasts in a single click.
   *
   * Defaults to 3. Set to `0` to disable to the button entirely.
   */
  showClearAllButtonAt?: number;
```

and then later down below in the props destructuring/defaults, we'd do:
```
  showClearAllButtonAt = 3,
```

Thoughts? Worth doing or nah?

Reply:
@cee-chen I like this approach especially that it consolidates 2 concerns, but I'd also like to point out that by adopting this pattern and setting the default to 3 we'd be making the "dismiss button" a first class functionality of the toast list, whilst this is fine for us (Kibana) it'd be a breaking change of some sort of other eui consumers.

Reply:
That been said, perhaps we set the default to `0` and have every consumer set their own threshold, what do you think?

Reply:
To be honest, I'd be totally fine to adopt this new pattern as a default. I think it's a very sensible one that I wish other apps had (I use an online productivity app called Habitica that generates 3 toasts per action, and I can sometimes sit there clicking dismissing toasts for a minute straight 🥲).

Other EUI consumers can simply disable the new feature in less than one line if they don't want it, so I think it's fairly safe to add. @andreadelrio, any opinions here?

Reply:
I like the idea of making it the default, seems like a nice UX consideration that's easy to disable if needed.